### DISCUSSION TITLE
Globally Darken Shadows via MantineProvider

### DISCUSSION THREAD

**Discussion by JasonWeise** [mantinedev/mantine#660] [Q&A] [answered]
I just have to say I am absolutely loving this library and has now become my one and only UI library for React, amazing work!.

I just have a question regarding being able to globally set the darkness of the box shadow for components like Paper.
With a current project I am working on the color scheme means the current default shadow opacity of (I think) 5% in Mantine means any drop shows are almost mperceivable. If I manually set the shadow opacity in the browser dev tools to 15% it looks great.

Is there a way to set the shadow opacity via ManrineProvider like there is for colors so this will apply across the entire app?

Thanks in advance!

**Comment by rtivital** [comment, marked as answer]
Hi Jason, thanks for kind words! You can define shadows on theme, property is called `shadows` – https://mantine.dev/theming/extend-theme/#spacing-radius-and-shadows

**Comment by JasonWeise** [comment]
OMG! I cannot believe I missed that, I scrolled past it like 5 times :/

Thanks so much for your help!

**Comment by ludwignagelhus** [comment]
This page is giving me a 404 now. What is the new url?

**Reply by rtivital** [reply]
https://mantine.dev/theming/theme-object/
